views/Account.react.js

```diff
@@ -92,17 +93,17 @@ class AccountPage extends Component {
         }
         return response.json()
       })
-      .then((response) => {
+      .then(({ data }) => {
         this.setState({
-          _id: response['_id'],
-          uid: response['uid'],
-          display_name: response['display_name'],
-          title: response['title'],
-          department: response['department'],
-          company: response['company'],
-          mail: response['mail'],
-          ldap: response['ldap'],
-          icon: response['icon'],
+          _id: data['_id'],
+          uid: data['uid'],
+          display_name: data['display_name'],
+          title: data['title'],
+          department: data['department'],
+          company: data['company'],
+          mail: data['mail'],
+          ldap: data['ldap'],
+          icon: data['icon'],
```

Comment:
should we be setting `this.state.user` to the `data` returned from this endpoint? Then we could remove all these other state attributes because we could get them from the user object. It would also improve the `editUserInfo` function because we could use the state value instead of making a new `user` constant.

Reply:
This is outside of the scope of this ticket